heareval/evaluation/runner.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+"""
+Performs evaluation on embedding predictions.
+"""
+
+import json
+from pathlib import Path
+
+import click
+from tqdm import tqdm
+
+from heareval.evaluation.task_evaluation import task_evaluation
+
+
+@click.command()
+@click.option(
+    "--embeddings-dir",
+    default="embeddings",
+    help="Location of task embeddings to compute evaluation on.",
+    type=click.Path(exists=True),
+)
```

Comment:
Wait why don't we need the package we're evaluating here? hearbaseline etc.

Reply:
Why not compute evaluation on all them at once? They are all subfolders in the embeddings dir. Can add an option to specify and just run one.

Reply:
True.